requirements/edx/github.in

```diff
@@ -63,7 +63,9 @@ git+https://github.com/edx/MongoDBProxy.git@d92bafe9888d2940f647a7b2b2383b29c752
 git+https://github.com/edx/django-require.git@0c54adb167142383b26ea6b3edecc3211822a776#egg=django-require==1.0.12
 
 # Our libraries:
--e git+https://github.com/openedx/blockstore.git@1.2.1#egg=blockstore==1.2.1  # Note: Blockstore 1.2.2 is failing.
+# FIXME update when https://github.com/openedx/blockstore/pull/186 merged/tagged
+#-e git+https://github.com/openedx/blockstore.git@1.2.4#egg=blockstore==1.2.4  # Note: Blockstore 1.2.2 is failing.
+-e git+https://github.com/open-craft/blockstore.git@jill/blockstore-1.2.4#egg=blockstore==1.2.4
```

Comment:
**TODO** Merge and tag https://github.com/openedx/blockstore/pull/186, then update the dependency to use the new tag.